### ISSUE TITLE
Adjust colors for ride calendar theme that do not match accessibility guidelines

### ISSUE THREAD

**Issue by navaronbracke** [navaronbracke/weforza#382]
The ride calendar theme does not match the WCAG 2.1 AAA guideline for UI components.
Adjust the colors so they match the guideline.

https://accessibleweb.com/color-contrast-checker/

We should be using a tetradic color scheme, and use filled decorations everywhere (the contrast between filled & outlined does not work as well as expected)

We should apply a tetradic color scheme for light & dark mode, for both Android and iOS.
The starting point would be the primary color in each theme. (colorScheme.primary on Android & CupertineColors.activeBlue on iOS)

1) Pick the starting color (in hexadecimal) and change the color mode to Hue, Saturation, Brightness
2) Copy the color 4 times (as the palette is 4 colors)
3) The first color stays unchanged
4) The second color should shift its hue by 30 points
5) The third color should shift its hue by 180 points
6) The fourth color should shift its hue by 180 + 30 points
